pages/apps/index.tsx

```diff
@@ -13,15 +17,60 @@ const feature = config.features.marketplace;
 const Marketplace = dynamic(() => import('ui/pages/Marketplace'), { ssr: false });
 
 const Page: NextPage = () => {
+  const isMobile = useIsMobile();
+  const links = [];
+
+  if (feature.isEnabled) {
+    if (feature.submitFormUrl) {
+      links.push({
+        label: 'Submit app',
+        href: feature.submitFormUrl,
+        icon: 'plus' as IconName,
+      });
+    }
+    if (feature.suggestIdeasFormUrl) {
+      links.push({
+        label: 'Suggest ideas',
+        href: feature.suggestIdeasFormUrl,
+        icon: 'edit' as IconName,
+      });
+    }
+  }
+
   return (
     <PageNextJs pathname="/apps">
       <>
         <PageTitle
           title="DAppscout"
-          contentAfter={ feature.isEnabled && (
-            <LinkExternal href={ feature.submitFormUrl } variant="subtle" fontSize="sm" lineHeight={ 5 } ml="auto">
-              Submit app
-            </LinkExternal>
+          contentAfter={ (isMobile && links.length > 1) ? (
```

Comment:
Consider moving the `PageTitle` and its logic into the `Marketplace` component, following the pattern used for other pages. It is not necessary to fix this immediately; it can be done in a separate pull request later.